eZ/Publish/Core/REST/Server/Output/ValueObjectVisitor/RestExecutedView.php

```diff
@@ -34,16 +35,22 @@ class RestExecutedView extends ValueObjectVisitor
      */
     protected $contentService;
 
+    /** @var \eZ\Publish\Core\Repository\LocationResolver\LocationResolver */
+    protected $locationResolver;
+
     /**
      * @param \eZ\Publish\API\Repository\LocationService $locationService
      * @param \eZ\Publish\API\Repository\ContentService $contentService
+     * @param \eZ\Publish\Core\Repository\LocationResolver\LocationResolver $locationResolver
      */
     public function __construct(
         LocationService $locationService,
```

Comment:
location service don't seems to be used anymore, or is it used by parent?

Reply:
@andrerom yes, it's not used anywhere, removed in: https://github.com/ezsystems/ezpublish-kernel/pull/3055/commits/e4a56124c49504625c6e1f3a69295fda46bcbfd4